fabric-transaction-api-v1/src/main/java/net/fabricmc/fabric/api/transaction/v1/ChangelistDataKey.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.transaction.v1;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A {@link TransactionDataKey} that implements a simple change list. Each
+ * transaction holds parts of that change list, the items of the list describe
+ * how to modify the backing object, e.g: "insert 10 buckets of water"
+ *
+ * @param <T> the change list item type
+ */
+public interface ChangelistDataKey<T> extends TransactionDataKey<List<T>> {
```

Comment:
`ChangelistDataKey` -> `ChangeListDataKey`

Reply:
That works, though it's common to say "Changelist" as if it were a word of its own, so I can see where that comes from.